pilot/pkg/model/service.go

```diff
@@ -846,6 +847,23 @@ func (u NoopAmbientIndexes) WorkloadsForWaypoint(scope WaypointScope) []*Workloa
 
 var _ AmbientIndexes = NoopAmbientIndexes{}
 
+type AddressInfo struct {
+	*workloadapi.Address
+}
+
+func (i AddressInfo) ResourceName() string {
+	var name string
+	switch addr := i.Type.(type) {
+	case *workloadapi.Address_Workload:
+		ii, _ := netip.AddrFromSlice(addr.Workload.Address)
+		name = addr.Workload.Network + "/" + ii.String()
+	case *workloadapi.Address_Service:
+		// TODO GregHanson currently not supported in ztunnel
+		name = addr.Service.Namespace + "/" + addr.Service.Hostname
```

Comment:
since this is not currently supported, does using this here cause issues? i would expect so.. maybe I should add support for this since we will need a primary xds key (services currently support several addresses)

Reply:
yeah, Service does not have a network field, so "network/address" does not work here

Reply:
yeah I think this is a removal bug with services in my ztunnel PR, im going to address this. i don't think any changes are required here but im surprised ci did not catch this in ztunnel or istio. going to add tests for this too in ztunnel